minard/templates/channel_database.html

```diff
@@ -30,7 +30,6 @@
                     <option value="16">16</option>
                     <option value="17">17</option>
                     <option value="18">18</option>
-                    <option value="19">19</option>
```

Comment:
I don't use it much any more, but I used to test things out by using the teststand database which did have data for crate 19. I think this is relatively harmless so maybe we should just keep it on the off chance that someone wants to look at data from the teststand later.